index.d.ts

```diff
@@ -271,6 +316,21 @@ declare module 'warframe-items' {
         locTag?: string;
     }
 
+    enum ProductCategory {
```

Comment:
not sure that this is correct, I think this would be a type like the others below it... I don't think they're accessible as you're using them here. maybe I don't fully understand typing...

Reply:
It was meant to be an union type but apparently quicktypes generated it wrongly, we can just remove it.